### DISCUSSION TITLE
The directory you are trying to deploy doesn't exist

### DISCUSSION THREAD

**Discussion by AliMurtazaJaffery** [JamesIves/github-pages-deploy-action#789] [Q&A] [answered]
<img width="1440" alt="Screenshot 2021-07-03 at 4 47 52 AM" src="https://user-images.githubusercontent.com/60510673/124326990-fb828600-dbb9-11eb-9694-8c029e56cbf8.png">

<img width="1440" alt="Screenshot 2021-06-21 at 3 09 03 PM" src="https://user-images.githubusercontent.com/60510673/124327184-44d2d580-dbba-11eb-8f1c-fb91352e692c.png">

Any form of help will be appreciated.

**Comment by JamesIves** [comment, marked as answer]
The folder you're trying to deploy does not exist, whatever is being generated from your build task is not a folder called "build".

**Reply by AliMurtazaJaffery** [reply]
Yeah I was using Next.js and for that I needed the out folder.
Thanks!
